engine/installation/windows/docker-ee.md

```diff
@@ -114,16 +114,16 @@ installs, or install on air-gapped systems.
 
 3.  Test your Docker EE installation by running the `hello-world` container.
 
-    ```ps
+    ```PowerShell
     PS> docker container run hello-world:nanoserver
     ```
 
 ## Install a specific version
 
 To install a specific Docker version, you can use the `MaximumVersion` and `MinimumVersion` flags. For example:
 
-```ps
-Install-Package -Name docker -ProviderName DockerMsftProvider -Source Docker -Force -MaximumVersion 17.03
+```PowerShell
+Install-Package -Name docker -ProviderName DockerProvider -Source Docker -Force -MaximumVersion 17.03
```

Comment:
I notice in the examples above we have a `PS>` prompt, and not here (and below); is that deliberate?

Reply:
I think we should remove them
everywhere since they make copy pasting impossible